postfixCalculator/postfixCalculator/stack.c

```diff
@@ -0,0 +1,71 @@
+#include "stack.h"
+
+#include <stdio.h>
+#include <malloc.h>
+#include <stdbool.h>
+
+typedef struct {
```

Comment:
```suggestion
typedef struct Node {
```
Потому что на 9-й строчке декларируется использование структуры Node, а структуру с таким именем никто не объявляет, есть только typedef. Подозреваю, что это приводит к предупреждению компилятора (хотя не удивлюсь, если нет, но всё равно выглядит пугающе).